tests/performance/src/main/java/org/mule/runtime/core/policy/DefaultPolicyStateHandlerBenchmark.java

```diff
@@ -19,17 +19,17 @@
 import org.mule.runtime.core.internal.policy.DefaultPolicyStateHandler;
 import org.mule.runtime.core.privileged.event.BaseEventContext;
 
-import java.util.Optional;
-
 import org.apache.commons.lang3.tuple.Pair;
 import org.openjdk.jmh.annotations.Benchmark;
 import org.openjdk.jmh.annotations.Threads;
 
+import java.util.Optional;
+
 public class DefaultPolicyStateHandlerBenchmark extends AbstractBenchmark {
 
-  private PolicyStateHandler handler = new DefaultPolicyStateHandler();
+  private final PolicyStateHandler handler = new DefaultPolicyStateHandler();
 
-  private Processor dummyProcessor = event -> event;
+  private final Processor dummyProcessor = event -> event;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this final field static too. ([squid:S1170](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1170))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7617&issues=AWhi1UbNotIXnmsUbHE5&open=AWhi1UbNotIXnmsUbHE5)

[ISSUE_KEY]: AWhi1UbNotIXnmsUbHE5